test/tensor/Jamfile

```diff
@@ -46,5 +48,13 @@ test-suite boost-ublas-tensor-test
           test_tensor_matrix_vector.cpp
           test_static_extents.cpp
           test_static_strides.cpp
+        #   test_fixed_rank_operators_arithmetic.cpp
+        #   test_static_operators_arithmetic.cpp
+        #   test_fixed_rank_operators_comparison.cpp
+        #   test_static_operators_comparison.cpp
+        #   test_fixed_rank_expression_evaluation.cpp
+        #   test_static_expression_evaluation.cpp
```

Comment:
is there a reason why the tests are commented?

Reply:
My computer was running out of memory if I uncomment them. Therefore I don't know appveyor or travis will do the same.

Reply:
It was giving me length exception for vector when resize.

Reply:
Okay I will take a look at the performance issues.

Reply:
I'm checking at my end if there is a bug or is it a memory issue.

Reply:
Compiling with multiple cores, i.e. `-j8` I get:
```bash
g++-7: internal compiler error: Killed (program cc1plus)
Please submit a full bug report,
with preprocessed source if appropriate.
See <file:///usr/share/doc/gcc-7/README.Bugs> for instructions.
```
Using only two cores, cc1plus is killed when compiling 
```bash 
test_fixed_rank_operators_comparison.cpp
```
or
```bash 
test_fixed_rank_operators_arithmetic.cpp
```
I think it might help to instantiate less templates.

Reply:
I think I found the problem. It is giving error because defining fixture before BOOST_AUTO_TEST_CASE.

Reply:
I just tested it and it's working fine now.